data/json/construction.json

```diff
@@ -7867,5 +7867,62 @@
     "components": [ [ [ "rock_large", 22 ] ] ],
     "pre_terrain": "t_dirt",
     "post_terrain": "t_rockyobstacle"
+  },
+  {
+    "type": "construction",
+    "id": "constr_brick_oven_prep",
+    "group": "build_brick_oven",
+    "category": "CONSTRUCT",
+    "required_skills": [ [ "fabrication", 0 ] ],
```

Comment:
Should probably be at least fab 1

Reply:
Yeah good catch, making it fab 2. Nothing fancy but still some opportunities to screw up.